### DISCUSSION TITLE
empty wire:loading.attr causes infinite round trips

### DISCUSSION THREAD

**Discussion by marco-silvestri** [livewire/livewire#5019] [7. Bugs (archived)]
Howdy!
Today I've encountered a weird non-bug: leaving the wire:loading.attr empty, it causes a huge amount of round trips from the server, because it cannot clearly understand which class to toggle.

Here a snippet of the culprit:
```
    <table wire:loading.attr="">
        <thead>
            {{ $thead }}
        </thead>
        <tbody wire:loading.remove>
            {{ $tbody }}
        </tbody>
    </table>
```

**Comment by joshhanley** [comment]
@marco-silvestri ah interesting! Yeah that's not ideal. Do you think you'd be able to submit a PR with a fix or at least a failing test?

Thanks!

**Reply by marco-silvestri** [reply]
Hi @joshhanley, I have an idea on how to do it. I'll take care of it before the week ends!

**Reply by joshhanley** [reply]
@marco-silvestri ok great, thanks! 🙂

**Comment by marco-silvestri** [comment]
Hi @joshhanley! So, I have been playing with a simple test, and it seems that if there is only one component, there is nothing weird in its behaviour. I suspect things go crazy when the page is more complex/has more components. I'll carry more tests and give you an update!

**Reply by joshhanley** [reply]
@marco-silvestri ok good to know! If you do find out exactly what triggers it, we can look further into why it's happening 🙂
